cv32e40p/docs/VerifPlans/README.md

```diff
@@ -18,66 +25,88 @@ Under the heading `Review` is one of following:
+### Base instruction set plus standard instruction extensions
 
+_Refer to the VerifPlans/ISA/RV32/Simulation directory for specific Verification Plan status for each supported extension._
 ### Interrupts
 
 | Feature | Capture | Review | Comment |
 |---------|---------|--------|---------|
-| CLINT | Captured (**updated**) | Ready for Review | |
-| CLIC | N/A | N/A | Not a CV32E40P Feature |
+| CLINT | Captured | Complete | |
+| CLIC | | | Not a CV32E40P Feature |
 
 ### Debug & Trace
 
 | Feature | Capture | Review | Comment |
 |---------|---------|--------|---------|
-| Debug | Captured  (**updated**) | Ready for Review | |
-| Trigger module | Captured (v1) | Complete | Not a CV32E40P Feature |
+| Debug | Captured | Complete | |
+| Trigger module | Captured | Complete | Not a CV32E40P Feature |
 | Tracer | N/A | N/A | Behavioral model, not RTL |
 
 ### Privileged spec
 
 | Feature | Capture | Review | Comment |
 |---------|---------|--------|---------|
-| CSRs | Captured (**formal**) | Not Available for Review | Verified by formal |
+| CSRs | Incomplete | | |
 | User mode | N/A| N/A | Not a CV32E40P Feature |
 | PMP | N/A | N/A | Not a CV32E40P Feature |
 
 ### Micro-architecure
 
 | Feature | Capture | Review | Comment |
 |---------|---------|--------|---------|
-| OBI     | Captured (v1) | Reviewed | |
-| Sleep Unit | Captured (v1) | Reviewed | Updates pending based on review feedback |
-| Pipelines | Captured (v1) | Reviewed | Updates pending based on review feedback|
-| FPU File Register | Captured (**new**) | Ready for Review | |
-
-### F and Zfinx extensions
-**Note**: As verifying all floating points instructions using formal tools **only** is too complex and would have required too much processing power, the missing features of F/Zfinx extension have been verified by simulation. 
-
-| Features | Capture | Review | Comment |
-|---------|---------|--------|---------|
-| DIV and SQRT Instructions | Captured (**new**) | Ready for Review | Not covered at all by formal verification |
-| FMUL/FDIV | Captured (**new**) | Ready for Review | Formal coverage issue: some bits of operands have been tied low | 
-| FNMADD/FMADD | Captured (**new**) | Ready for Review | Formal coverage issue: some bits of operands have been tied low | 
-| FNMSUB/FMSUB | Captured (**new**) | Ready for Review | Formal coverage issue: some bits of operands have been tied low |
-| Other F instructions | Captured (**new**) | Ready for Review | Formal pipeline issue: Add preceeding F multicycle instruction |
+| OBI     | Complete | Reviewed | |
+| Sleep Unit | Complete | Reviewed | Updates pending based on review feedback |
+| Pipelines | Complete | Reviewed | Updates pending based on review feedback|
 
 ### Xpulp instruction extensions
-**Note**: Xpulp instructions, as other instructions, has been mainly verified by formal tools. Some aspects needed a complementary simulation verification
+**Note**: Xpulp instructions are "exercised, but not fully verified" in CV32E40P.
```

Comment:
Is that true?  It was my understanding that as far as possible, Xpulp instructions were fully verified in formal and any Xpulp instructions not fully verified in formal would be fully verified in simulation.

Reply:
When talking about partly verified Pulp instructions, it is mainly about multiplication operands for either scalar or SIMD instructions.
Same case for Floating-Point multiplications.

Reply:
I restored the v1 file as it was, and it was saying "Xpulp instructions are "exercised, but not fully verified" in CV32E40P."

The part you mention is about v1 verification **only**.

For v2, is it exactly as Pascal said

Reply:
Interesting...  I have used similar strategies in the past and consider that to be "fully verified".  There is no need to verify that a specific operation is correct for all possible input values.  Your strategy to constrain bits [29:02] to either 28'b0 or 28'b1 is perfectly valid and sufficient to call the operation "fully verified".

Reply:
Hi @MikeOpenHWGroup 
I agree with you for the general case of 32 bits scalar integer instructions.
It is not fully verified for SIMD operations. I would like to have this free bits strategy applied depending on data size.
So when cutting the 32 bits in 2 or 4 independent pieces, having free bits on each part would allow full verification.